pkg/sql/pg_catalog_test.go

```diff
@@ -28,32 +28,77 @@
 package sql
 
 import (
+	"bufio"
 	"context"
 	"encoding/json"
 	"flag"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
 	"testing"
 
 	"github.com/cockroachdb/cockroach/pkg/base"
+	"github.com/cockroachdb/cockroach/pkg/sql/types"
 	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
 	"github.com/cockroachdb/errors/oserror"
+	"github.com/lib/pq/oid"
 )
 
 // Test data files
 const (
-	pgCatalogDump = "pg_catalog_tables.json"              // PostgreSQL pg_catalog schema
-	expectedDiffs = "pg_catalog_test_expected_diffs.json" // Contains expected difference between postgres and cockroach
-	testdata      = "testdata"                            // testdata directory
+	pgCatalogDump   = "pg_catalog_tables.json"              // PostgreSQL pg_catalog schema
+	expectedDiffs   = "pg_catalog_test_expected_diffs.json" // Contains expected difference between postgres and cockroach
+	testdata        = "testdata"                            // testdata directory
+	catalogPkg      = "catalog"
+	catconstantsPkg = "catconstants"
+	constantsGo     = "constants.go"
+	vtablePkg       = "vtable"
+	pgCatalogGo     = "pg_catalog.go"
+)
+
+// strings used on constants creations and text manipulation
+const (
+	pgCatalogPrefix          = "PgCatalog"
+	pgCatalogIDConstant      = "PgCatalogID"
+	tableIDSuffix            = "TableID"
+	tableDefsDeclaration     = `tableDefs: map[descpb.ID]virtualSchemaDef{`
+	tableDefsClosure         = `},`
```

Comment:
Rename to tableDefCloseBracket (not sure Closure is the right word here)

Reply:
I named it closure because I was thinking of the function "printsBeforeClosure" probably closure is not the right word,
What printsBeforeClosure does:
It will read and skips all the lines (Except comments) until find the "closure" line, then it will print the s text and the "closure" text.

Please let me know if you believe there is a better word for this.